### ISSUE TITLE
How to export ascii-art video locally？

### ISSUE THREAD

**Issue by 8824PPZ** [mariiaan/cmdplay-pp#5]

**Comment by mariiaan** [comment]
nice feature idea, will work on it in a few days :)

**Comment by 8824PPZ** [comment]
> nice feature idea, will work on it in a few days :)

OK. Thanks.

**Comment by mariiaan** [comment]
question, would it better to have an export as ASCII characters (like to a .txt file) or as an video with ascii art (like if you'd record it)?

**Comment by 8824PPZ** [comment]
> question, would it better to have an export as ASCII characters (like to a .txt file) or as an video with ascii art (like if you'd record it)?

I think it’s better to export an video with ascii art. Because video is more interesting than text.

**Comment by its-19818942118** [comment]
hmmm well i think how about adding both a txt and video option. cuz i think its better in terms of perfomance or so i belive as a terminal user. 😄

**Comment by mariiaan** [comment]
Yeah you're probably right. The export as txt is easy, however the video option is a bit more involved.

If you take a look at the "export-asciiart" branch, you'll see that I've already started writing a software text renderer and video encoder.

My main concern was that I didn't want to add a huge list of dependencies so I've decided to go that way.

Unfortunately, I'm really busy with other projects at the moment. I'll still try to contribute every now and then.

**Comment by its-19818942118** [comment]
well yeah even though the video option is a bit involved its univrsally availabe and easy to use. but txt will have some limitations on rendering or stuff if no work is involved my opinion as a newbie in the world of coding and programming. os yeah for each frame then etc. but  hey i maybe wrong. do let me know. also thanks for your work my idea was exaactly like this. glad to see it in motion 😄

**Comment by Sandesh13fr** [comment]
how do I use this.

**Comment by its-19818942118** [comment]
> how do I use this.

@Sandesh13fr hmm its not a feature implmentation yet

**Comment by its-19818942118** [comment]
hmm ok so i think i might have um find an alternative solution... (linux only i think) so instead of local export. we can use [***Asciinema***](https://asciinema.org/). it just works hmmm. so um its not really like an local export.. i just remembered about this. been a long time so... you can check em out
